doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -295,7 +295,285 @@ Your output should look like the following:
+
+Your output should look like the following:
+
+::
+
+    batchtransformjobs.sagemaker.aws.amazon.com         2019-11-20T17:12:34Z
+    endpointconfigs.sagemaker.aws.amazon.com            2019-11-20T17:12:34Z
+    hostingdeployments.sagemaker.aws.amazon.com         2019-11-20T17:12:34Z
+    hyperparametertuningjobs.sagemaker.aws.amazon.com   2019-11-20T17:12:34Z
+    models.sagemaker.aws.amazon.com                     2019-11-20T17:12:34Z
+    trainingjobs.sagemaker.aws.amazon.com               2019-11-20T17:12:34Z
+
+Ensure that the operator pod is running successfully. Use the following
+command to list all pods:
+
+::
+
+    kubectl -n sagemaker-k8s-operator-system get pods
+
+You should see a pod
+named \ ``sagemaker-k8s-operator-controller-manager-*****`` in the
+namespace \ ``sagemaker-k8s-operator-system``  as follows:
+
+::
+
+    NAME                                                         READY   STATUS    RESTARTS   AGE
+    sagemaker-k8s-operator-controller-manager-12345678-r8abc   2/2     Running   0          23s
+
+​
+
+Namespace-scoped deployment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You have the option to install your operator within the scope of an individual Kubernetes namespace. In this mode, the controller will only monitor and reconcile resources with Amazon SageMaker if the resources are created within that namespace. This allows for finer grained control over which controller is managing which resources. This is useful for deploying to multiple AWS accounts or controlling which users have access to particular jobs.
+
+This guide outlines how to install an operator into a particular, predefined namespace. To deploy a controller into a second namespace, follow the guide from beginning to end and change out the namespace in each step.
+
+
+
+
+Create an OpenID Connect Provider for Your EKS Cluster
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following instruction will create and associate an OIDC provider
+with your EKS cluster.
+
+Set the local ``CLUSTER_NAME`` and \ ``AWS_REGION`` environment
+variables as follows:
+
+::
+
+    # Set the Region and cluster
+    export CLUSTER_NAME="<your cluster name>"
+    export AWS_REGION="<your region>"
+
+Use the following command to associate the OIDC provider with your
+cluster. For more information, see \ `Enabling IAM Roles for Service
+Accounts on your
+Cluster. <https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html>`__
+
+::
+
+    eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
+        --region ${AWS_REGION} --approve
+
+Your output should look like the following:
+
+::
+
+    [_]  eksctl version 0.10.1
+    [_]  using region us-east-1
+    [_]  IAM OpenID Connect provider is associated with cluster "my-cluster" in "us-east-1"
+
+Now that the cluster has an OIDC identity provider, you can create a
+role and give a Kubernetes ServiceAccount permission to assume the role.
+
+Get your OIDC ID
```

Comment:
Here we say `Get your` but in the previous example we used `Get the`. Same with the IAM role, the phrasing is different. Also the same with the `EKS Cluster` vs `Cluster`.

Reply:
@RedbackThomson I made minor adjustments to some titles because the build system doesn't support any duplicated titles. It creates links based on the titles, so there would be two sections with identical links. I'm definitely open to any suggestions on how to update the titles to avoid this duplication